flexmeasures/data/schemas/scheduling/storage.py

```diff
@@ -76,10 +78,17 @@ class StorageFlexModelSchema(Schema):
     soc_min = fields.Float(validate=validate.Range(min=0), data_key="soc-min")
     soc_max = fields.Float(data_key="soc-max")
 
-    power_capacity_in_mw = QuantityField(
+    power_capacity_in_mw = QuantityOrSensor(
```

Comment:
@victorgarcia98 I don't think we actually support passing this as a sensor ID. It looks like in `MetaStorageScheduler._prepare` we only support a `Quantity` right now:

https://github.com/FlexMeasures/flexmeasures/blob/8b25ae9951d950ceb1f904ace388966faf0ad25d/flexmeasures/data/models/planning/storage.py#L114-L134

Reply:
Oh, you're right. 

Should a hotfix (using the field type `QuantityField`) follow? Let me create an issue to discuss this there.

Reply:
https://github.com/FlexMeasures/flexmeasures/issues/984